config.toml

```diff
@@ -1,7 +1,7 @@
 canonifyurls = true
-title = "The USGS OWI blog"
+title = "Water Data For The Nation News"
 name = ["Laura DeCicco"]
```

Comment:
I wonder if this is the appropriate name and email going forward. Seems like this might be within the Web Communications Branch, probably a question for Jim or Brad.